hypervisor/lib/crypto/mbedtls/config.h

```diff
@@ -2605,7 +2605,7 @@
  *            on it, and considering stronger message digests instead.
  *
  */
-#define MBEDTLS_SHA1_C
+//#define MBEDTLS_SHA1_C
```

Comment:
for coding style, should we use "//"?

Reply:
The file is copied from mBedtls lib, "//" is the original coding style to comment code line in this file.

Reply:
make sure check patch is fine.